cli/collection_create.go

```diff
@@ -16,34 +16,41 @@ import (
 
 	"github.com/spf13/cobra"
 
+	acpIdentity "github.com/sourcenetwork/defradb/acp/identity"
 	"github.com/sourcenetwork/defradb/client"
 )
 
 func MakeCollectionCreateCommand() *cobra.Command {
+	const identityFlagLongRequired string = "identity"
+	const identityFlagShortRequired string = "i"
+
+	var identityValue string
 	var file string
+
 	var cmd = &cobra.Command{
-		Use:   "create <document>",
+		Use:   "create [-i --identity] <document>",
```

Comment:
todo: It will be tedious for users to have to supply an identity param to every command, especially when authentication arrives - can we not do something similar to git and other such IDed CLIs and have a `userCfg` file somewhere for them?

Reply:
Is it optional in this PR? Users will not be forced to supply it?

Reply:
Yes `--identity` is optional (only place it is mandatory on is the `defradb client acp policy add` command).

Note: all functionality that existed before this PR, should continue to behave in the same manner

Reply:
Nice - thanks :)  Happy for the `userCfg` file stuff to be done outside of this PR then, creating a ticket now might be handy though

Reply:
- [ ]  Make issue for adding user config or add as a subtask to auth task

Reply:
Issue: https://github.com/sourcenetwork/defradb/issues/2416